projects/1.20/assets/food-expansion-reimagined/foodexpansion/lang/zh_cn.json

```
@@ -1,28 +1,28 @@
 {
-    "item.foodexpansion.jelly": "翡翠果冻",
+    "item.foodexpansion.jelly": "绿色果冻",
     "item.foodexpansion.bacon": "培根",
     "item.foodexpansion.cooked_bacon": "香脆培根",
     "item.foodexpansion.fried_egg": "煎蛋",
     "item.foodexpansion.bacon_and_egg": "培根煎蛋",
-    "item.foodexpansion.carrot_seed_soup": "胡萝卜浓汤",
-    "item.foodexpansion.squid": "生鱿鱼",
-    "item.foodexpansion.cooked_squid": "烤鱿鱼",
-    "item.foodexpansion.compressed_flesh": "腊肉",
-    "item.foodexpansion.chocolate_bar": "巧克力棒",
-    "item.foodexpansion.spider_soup": "蛛眼浓汤",
+    "item.foodexpansion.carrot_seed_soup": "胡萝卜汤",
+    "item.foodexpansion.squid": "生鱿鱼片",
+    "item.foodexpansion.cooked_squid": "熟鱿鱼片",
+    "item.foodexpansion.compressed_flesh": "肉干",
+    "item.foodexpansion.chocolate_bar": "巧克力",
+    "item.foodexpansion.spider_soup": "蜘蛛眼汤",
     "item.foodexpansion.nether_wart_soup": "下界疣煲",
     "item.foodexpansion.cactus_fruit": "仙人掌果",
     "item.foodexpansion.horse_meat": "生马肉",
     "item.foodexpansion.cooked_horse_meat": "熟马肉",
     "item.foodexpansion.forbidden_fruit": "§e禁果",
     "item.foodexpansion.cooked_mushroom": "烤蘑菇",
     "item.foodexpansion.carrot_pie": "胡萝卜蛋糕",
-    "item.foodexpansion.bat_wing": "生蝙蝠翅膀",
-    "item.foodexpansion.cooked_bat_wing": "烤蝙蝠翅膀",
+    "item.foodexpansion.bat_wing": "生蝙蝠翼",
```

```suggestion
    "item.foodexpansion.bat_wing": "生蝙蝠翅",
```
原译其实没什么问题……例如`鸡翅`这种措辞是不是更好一点

---

这条如果要改则下面的`蝙蝠翼汤`也要改

话说剩余的这两条还要改吗
不改就可以合并了

---

这两条还要改吗

---

这条commit后合了吧